modules/data.atmosphere/R/download.Ameriflux.R

```diff
@@ -19,7 +19,7 @@ download.Ameriflux.site <- function(site_id) {
 ##' @param verbose should the function be very verbose
 ##' 
 ##' @author Josh Mantooth, Rob Kooper, Ankur Desai
-download.Ameriflux <- function(sitename, outfolder, start_date, end_date, overwrite=FALSE, verbose=FALSE) {
+download.Ameriflux <- function(sitename, outfolder, start_date, end_date, overwrite=FALSE, verbose=FALSE, ...) {
   # get start/end year code works on whole years only
   
   require(lubridate) #is this necessary?
```

Comment:
Got it - though I can't take the blame for this specific function, as I only added the ....  Found this helpful blog post http://yihui.name/en/2014/07/library-vs-require/ 

Catching a flight to Seattle in an hour, but will try to push changes when I can.